asf_heat_pump_suitability/pipeline/prepare_features/anchor_properties.py

```diff
@@ -0,0 +1,170 @@
+"""Module for identifying and analyzing potential anchor properties in LSOAs."""
+
+# TODO implement building footprint data for increased identification accuracy
+
+import logging
+from typing import Set
+from pathlib import Path
+
+import geopandas as gpd
+import pandas as pd
+
+from asf_heat_pump_suitability.getters.s3_getters import load_s3_data
+
+# Configure logging
+logger = logging.getLogger(__name__)
+
+# Configuration constants
+INPUT_CRS = "EPSG:4326"
+PROCESSING_CRS = "EPSG:27700"  # British National Grid
+
+# Set of anchor property types
+ANCHOR_PROPERTIES = {
+    # TODO refine
+    # alternate_categories for each type need to be further investigated to ensure that only the relevent building types are selected
+    "hospital",
+    "high_school",
+    "university_housing",
+    "college_university",
+    "public_service_and_government",
+    "central_government_office",
+    "government_services",
+    "local_and_state_government_offices",
+    "sports_and_recreation_venue",
+    "water_park",
+    "ice_skating_rink",
+    "swimming_pool",
+    "jail_and_prison",
+    "shopping_center",
+    "airport_terminal",
+    "airport",
+    "superstore",
+    "department_store",
+}
+
+
+def _safe_join_categories(categories: pd.Series) -> str:
```

Comment:
```suggestion
def _safe_join_str_categories(categories: pd.Series) -> str:
```
Here and in other functions, please can we add the return type to the name. The naming convention of functions for the repo is _action_returnType_descriptor_. Makes it easier to immediately know what the function does.